docs/section5/Rule5-8.md

```diff
@@ -0,0 +1,45 @@
+
+# Envelope - Rule 5-8  
+
+**Rule ID:** 5-8  
+**Rule Description:**  Fenestration U-factors for new buildings, existing buildings, and additions shall match the appropriate requirements in Tables G3.4-1 through G3.4-8 for the applicable glazing percentage for Uall.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** Tables G3.4-1 to G3.4-8  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+
+## Rule Logic:  
+
+- Get building climate zone: ```climate_zone = B_RMR.weather.climate_zone```  
+
+- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
+
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
+
+    - Get surfaces in space: ```for surface_baseline in space_baseline.surfaces:```  
+
+      - Check if surface is exterior wall: ```if ( surface_baseline.classification == "WALL" ) AND ( surface_baseline.adjacent_to == "AMBIENT" ): wall_surface_baseline = surface_baseline```  
+
```

Comment:
@JasonGlazer and @xingcx i believe we want to base this on the tilt of a surface to determine if a surface is a wall. If I remember correctly, we plan to flag 'tilt' as required for 90.1 so that the tilt is used instead of the surface classification.

Reply:
@xingcx Ok, the 90.1 definition for "wall" says that it is "60 degrees from horizontal or greater"